src/pages/overlay/components/buttons/Buttons.tsx

```diff
@@ -27,15 +28,30 @@ export default function Buttons<T extends ButtonsOptions = ButtonsOptions>(props
 
   return (
     <div className={styles.activationButtons}>
-      {optionsWithOnClick.map(option => (
-        <button
+      {optionsWithOnClick.map((option) => (
+        <Tooltip
           key={option.key}
-          onClick={option.onClick}
-          className={classes(option.active && styles.highlighted, option.type === "secondary" && styles.secondary)}
+          text={option.hoverText}
+          ariaLabel={option.title}
+          className={classes(
+            styles.tooltip,
+            option.active && styles.highlighted,
+            option.type === "secondary" && styles.secondary
+          )}
         >
-          <img src={option.icon} alt={option.title} />
-        </button>
+          <button
+            key={option.key}
+            onClick={option.onClick}
+            className={classes(
+              styles.btn,
+              option.active && styles.highlighted,
+              option.type === "secondary" && styles.secondary
```

Comment:
```suggestion
              option.active && styles.highlighted,
```
We only need to worry about the highlighting here